### ISSUE TITLE
rag-end2end-retriever Training Time Results

### ISSUE THREAD

**Issue by jamesoneill12** [huggingface/transformers#21820]
### Feature request

Hi, 

I would suggest including the training runtime for the new version of [RAG](https://github.com/huggingface/transformers/tree/main/examples/research_projects/rag-end2end-retriever) that is end-to-end. The hyperparameters for the results at the end of the README.md (under the "Comparison of end2end RAG (including DPR finetuning) VS original-RAG" section) are that below, but I've no idea the training time for these experiments on SQUAD. Currently my runtime on 2 GPUs is looking like months, so I want to know if there's something I've missed that causing training to be so slow or if it takes this long to run RAG end-to-end.  

--gpus 4 
--train_batch_size 4 
--eval_batch_size 
--max_source_length 128 
--max_target_length 25 
--val_max_target_length 25 
--test_max_target_length 25 
--label_smoothing 0.1 
--dropout 0.1 
--attention_dropout 0.1 
--weight_decay 0.001 
--adam_epsilon 1e-08 
--max_grad_norm 0.1 
--lr_scheduler polynomial 
--learning_rate 3e-05 
--num_train_epochs 10 
--warmup_steps 500 
--gradient_accumulation_steps 4
--distributed_retriever ray 
--num_retrieval_workers 4  

Thanks, James

### Motivation

Saves a lot of time before researchers decide if they've enough resources to work and build on this work or not. Also, suggestions on how to run training with less resources would also be useful. 

### Your contribution

Not sure it requires much help, apart from the original authors (or at least whoever ran the experiments corresponding to the results in the readme) including training time numbers in the readme results section.
